src/test/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerXMLInputFileTypeTest.java

```diff
@@ -0,0 +1,56 @@
+package edu.cornell.kfs.module.purap.batch.service.impl;
+
+import edu.cornell.kfs.module.purap.batch.JaggaerXMLInputFileType;
+import edu.cornell.kfs.sys.util.LoadSpringFile;
+import edu.cornell.kfs.sys.util.SpringEnabledMicroTestBase;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+@LoadSpringFile("edu/cornell/kfs/module/purap/jaggaer/xml/cu-spring-jaggaer-test.xml")
+public class JaggaerXMLInputFileTypeTest extends SpringEnabledMicroTestBase {
+    private static final String EXPECTED_FILE_EXTENSION_XML = "xml";
+    private static final String EXPECTED_AUTHOR_NAME_AJD299 = "ajd299";
+    private static final String EXAMPLE_FILENAME = "jaggaerSupplierUploadFile_ajd299_20231025_092052250.xml";
+    private static final String EXPECTED_FILENAME_PREFIX = "jaggaerSupplierUploadFile_ajd299_";
+    private static final String JAGGAER_STAGING_RELATIVE_PATH = "purap/jaggaer/xml";
+    private static final String PRINCIPAL_NAME_AJD299 = "ajd299";
+
+    private JaggaerXMLInputFileType jaggaerXMLInputFileType;
+
+    @Override
+    @BeforeEach
```

Comment:
Because this test class is currently using JUnit-4-related classes and annotations, please use the `@Before` annotation instead for consistency, unless it's causing problems.